static/js/tools/scatter2/info.tsx

```diff
@@ -37,9 +37,96 @@ function Info(): JSX.Element {
           to choose from, arranged in a topical hierarchy.
         </li>
       </ol>
-
-      {/* TODO(intrepiditee): Add descriptions examples */}
-
+      <p>Or you can start your exploration from these interesting points ...</p>
+      <ul>
+        <li>
+          <b>Asians Per Capita vs Median Income</b> for counties in
+          <a
+            href={
+              "#%26svx%3DMedian_Income_Person%26svpx%3D0-3%26svnx%3DMedian%20income%26svy%3DCount_Person_AsianAlone%26svpy%3D0-14-1%26svdy%3DCount_Person%26svny%3DAsian%20Alone%26pcy%3D1%26epd%3DgeoId%2F06%26epn%3DCalifornia%26ept%3DCounty"
```

Comment:
I think you can use the regular (no-encoded) chars in the url here. That would be easy for maintain.

Reply:
done